src/main/resources/assets/securitycraft/lang/fr_fr.json

```diff
@@ -0,0 +1,767 @@
+	"messages.securitycraft:irc.connected": "Le chat IRC n’est plus pris en charge à partir de la v1.8.3 de SecurityCraft. Si vous avez besoin d’aide, veuillez vous référer au canal #help sur SecurityCraft’s Discord!",
+	"messages.securitycraft:cageTrap.captured": "#player a été capturé à #location.",
+	"messages.securitycraft:frame.rightclick": "RClic droit avec un moniteur associé pour le voir.",
+	"messages.securitycraft:command.sc.usage": "Usage: /sc connect OU /sc disconnect OU /sc contact OU /sc resume",
+	"messages.securitycraft:naming.alreadyMatches": "Le nom de ce bloc correspond déjà ’#n’.",
+	"messages.securitycraft:naming.error": "Vous ne pouvez pas nommer ce bloc ’#n’, car le nom contient le caractère non valide ’#c’.",
+	"messages.securitycraft:thanks": "Merci d‘avoir utilisé SecurityCraft #!",
+	"messages.securitycraft:tip": "Conseil:",
+	"messages.securitycraft:tip.scHelp": "Tapez /sc help affichera le manuel SecurityCraft, qui affichera des informations d’aide pour les blocs / éléments SecurityCraft.",
+	"messages.securitycraft:tip.trello": "Consultez le tableau Trello pour SecurityCraft et signalez des bogues ou faites-nous des suggestions!",
+	"messages.securitycraft:tip.patreon": "Voulez-vous prendre en charge SecurityCraft?",
+	"messages.securitycraft:tip.discord": "Rejoignez notre serveur Discord pour discuter et recevoir des mises à jour et des annonces importantes concernant SecurityCraft!",
+	"messages.securitycraft:tip.scserver": "Le serveur officiel SecurityCraft est de retour ! Rejoignez-nous sur cette IP: 64.38.249.237",
+	"messages.securitycraft:universalKeyChanger.notKeypad": "Clic droit sur un clavier avec le changeur de code universel pour changer son mot de passe.",
+	"messages.securitycraft:universalKeyChanger.passcodeChanged": "Le code a été changé.",
+	"messages.securitycraft:passwordProtected.notSetUp": "Ce bloc n’a pas encore de mot de passe. Vous ne pouvez pas interagir avec.",
+	"messages.securitycraft:sentry.needsBlocBelow": "Les Sentinelles se placent sur un bloc!",
+	"messages.securitycraft:sentry.needsBlocsAround": "Assurez-vous que votre Sentinelle est entourée de blocs sur ses quatre côtés!",
+	"messages.securitycraft:sentry.mode1": "Mode actuel: agressif (toujours actif, attaque les mobs et les joueurs hostiles)",
+	"messages.securitycraft:sentry.mode2": "Mode actuel: Camouflage (Actif lorsqu’un joueur s’approche, attaque les joueurs) ",
+	"messages.securitycraft:sentry.mode3": "Mode actuel: inactif (jamais actif, n’attaque rien)",
+	"messages.securitycraft:blockpocket.multipleManagers": "Il y a plus d’un gestionnaire de Bloc de poche dans la structure.",
+	"messages.securitycraft:blockpocket.invalidBloc": "Bloc invalide \"%2$s\" à la position %1$s!",
+	"messages.securitycraft:blockpocket.invalidBloc.pillar": "Bloc invalide \"%2$s\" à la position %1$s! (Assurez-vous que le pilier tourne correctement!)",
```

Comment:
You accidentally removed the `k` from `Block` in these language keys. They should be `messages.securitycraft:blockpocket.invalidBlock` and `messages.securitycraft:blockpocket.invalidBlock.pillar`.

Reply:
Still needs to be changed.